CMakeLists.txt

```diff
@@ -433,7 +435,8 @@ if (NANOGUI_BUILD_PYTHON)
   include_directories("ext/pybind11/include" ${PYTHON_INCLUDE_DIR})
   add_library(nanogui-python-obj OBJECT
     python/main.cpp
-    python/constants.cpp
+    python/constants_glfw.cpp
```

Comment:
`python/constants.cpp` got split into two files.

1. `constants_glfw.cpp` has same as before, just in a different file.
2. `constants_entypo.cpp` is generated (which is why they got split)

Reply:
Good call